webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx

```diff
@@ -113,29 +114,30 @@ function AppTemplateCard(props) {
                         </Grid>
                         <Grid item xs={5} align="right">
                             <IconButton onClick={handleDialogOpen}>
-                                <Edit fontSize={"small"}/>
+                                <Edit fontSize={"small"} />
                             </IconButton>
                             <IconButton onClick={() => setOpenDeleteDialog(true)}>
-                                <Delete fontSize={"small"}/>
+                                <Delete fontSize={"small"} />
                             </IconButton>
                         </Grid>
                     </Grid>
                 </CardContent>
-                <Divider/>
+                <Divider />
                 <CardContent>
-                    <Grid container spacing={0} alignItems={"center"}>
+                    <Grid container spacing={0} alignItems={"left"}>
                         <Grid item sm={7}>
                             <Stack spacing={2}>
                                 <Typography variant="body2" color="text.secondary">
-                                    {appTemplate.description}
+                                    <Box>{appTemplate.description}</Box>
+                                    <Box>{t("appTemplate.configFile") + ": " + appTemplate.configFile}</Box>
```

Comment:
What do you want to accomplish here? I think a box represents a div container, which is not nice in a typography?

Reply:
I exactly want to have div-container ;-) Du kannst gern andere Vorschläge machen. Alles andere sah bei mir nicht schön aus.

Reply:
done